cmd/system-probe/config/adjust.go

```diff
@@ -47,6 +47,8 @@ func Adjust(cfg config.Config) {
 	adjustUSM(cfg)
 	adjustSecurity(cfg)
 
+	applyDefault(cfg, serviceDiscoveryNS("enabled"), true)
```

Comment:
Rather than this, you will want to declare the config value in `pkg/config/setup/system_probe.go` in `InitSystemProbeConfig`. You can set the default there.